inc/form.class.php

```diff
@@ -1071,18 +1071,18 @@ public function saveForm() {
       foreach ($found_questions as $id => $fields) {
          // If field was not post, it's value is empty
          if (isset($_POST['formcreator_field_' . $id])) {
-            $datas[$id] = is_array($_POST['formcreator_field_' . $id])
+            $datas['formcreator_field_' . $id] = is_array($_POST['formcreator_field_' . $id])
```

Comment:
this change and othrer ones in this file are required to ensure that call to PluginFormcreatorForms_answers::saveAnswers() receive always the same array structure in its first argument.